frontends/p4/fromv1.0/converters.h

```diff
@@ -158,11 +158,771 @@ class PrimitiveConverter {
     const IR::Statement *PrimitiveConverter_##NAME##_##__VA_ARGS__::convert(                    \
         ProgramStructure *structure, const IR::Primitive *primitive)
 
-// Is fed a P4-14 program and outputs an equivalent P4-16 program
-class Converter : public PassManager {
+///////////////////////////////////////////////////////////////
+
+class DiscoverStructure : public Inspector {
```

Comment:
I'm assuming this large change is just moving code from .cpp to .h without changing it at all.  Would be nice if github reviews gave a way of seeing the differences (if any) here.